### ISSUE TITLE
behave package needs parse_type >= 0.3.4

### ISSUE THREAD

**Issue by jchodera** [omnia-md/conda-recipes#246]
Anyone know anything about the `behave` package dependency `parse_type`?
```
BUILD START: behave-1.2.5-py27_0
Fetching package metadata: ......
Error: No packages found in current linux-64 channels matching: parse_type >=0.3.4
Removing old build directory
Removing old work directory
```

**Comment by rmcgibbo** [comment]
https://binstar.org/omnia/parse_type/files?

**Comment by jchodera** [comment]
Hopefully fixed by #247.
